src/Uno.UWP/Devices/Power/BatteryReport.Android.cs

```diff
@@ -0,0 +1,96 @@
+﻿
+using AndroidBatteryStatus = Android.OS.BatteryStatus;
+using UwpBatteryStatus = Windows.System.Power.BatteryStatus;
+
+
+namespace Windows.Devices.Power
+{
+	public sealed partial class BatteryReport
+	{
+		public int? ChargeRateInMilliwatts { get; }
+		public int? DesignCapacityInMilliwattHours { get; }
+		public int? FullChargeCapacityInMilliwattHours { get; }
+		public int? RemainingCapacityInMilliwattHours { get; }
+		public UwpBatteryStatus Status { get; }
+
+
+		internal BatteryReport()
+		{
+			// values are current for object creation
+
+			var ifilter = new Android.Content.IntentFilter(Android.Content.Intent.ActionBatteryChanged);
+			var batteryStatus = Android.App.Application.Context.RegisterReceiver(null, ifilter);
```

Comment:
As docs says: "Because it's a sticky intent, you don't need to register a [BroadcastReceiver] —by simply calling registerReceiver passing in null as the receiver as shown in the next snippet, the current battery status intent is returned"
https://developer.android.com/training/monitoring-device-state/battery-monitoring

Reply:
Makes sense, thanks.